lib/ReactViews/Preview/MappablePreview.jsx

```diff
@@ -39,12 +39,16 @@ class MappablePreview extends React.Component {
     t: PropTypes.func.isRequired
   };
 
-  async toggleOnMap(event) {
+  async toggleOnMap(_event) {
```

Comment:
Line 49/50 use `event`, what outer scope event is picked up by that after this change?